change/@fluentui-react-d832d1e9-a199-4dfc-a381-b0d33d1c2d16.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31253&commitId=0e41958e6f5c33443d21654286351558461683ab&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Split (compat)</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Split (compat).Primary Checked.hover main.chromium.png|2528| Changed|
|Button Split (compat).Primary Disabled.hover main.chromium.png|648| Changed|
|Button Split (compat).Primary with Primary Action Disabled.hover main.chromium.png|55| Changed|
|Button Split (compat).Primary with Primary Action Disabled.hover split.chromium.png|55| Changed|
|Button Split (compat).Primary.hover main.chromium.png|2157| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>